front/components/assistant/conversation/input_bar/editor/suggestion.ts

```diff
@@ -39,6 +39,11 @@ export function makeGetAssistantSuggestions() {
       const { suggestions, fallbackSuggestions } =
         editor.storage.MentionStorage.suggestions;
 
+      /// keeping the pre-defined order when queried without content
```

Comment:
is that in the scope of this PR? why introduce that?

Reply:
Yes, this is specifically the change that sorts the mentions by overriding the usual fuzzy/spreadLength search.

Reply:
I hope I didnt misunderstand #7709